docs/reference/commands/cluster.rst

```diff
@@ -0,0 +1,210 @@
+===========
+``cluster``
+===========
+
+Manage the MicroCeph cluster
+
+Usage:
+
+.. code-block:: none
+
+   microceph cluster [options]
+   microceph cluster [command]
+
+Available commands:
+
+.. code-block:: none
+
+  add         Generates a token for a new server
+  bootstrap   Sets up a new cluster
+  config      Manage Ceph Cluster configs
+  join        Joins an existing cluster
+  list        List servers in the cluster
+  migrate     Migrate automatic services from one node to another
+  remove      Removes a server from the cluster
+  sql         Runs a SQL query against the cluster database
+
+
+Global options:
+
+.. code-block:: none
+
+   -d, --debug       Show all debug messages
+   -h, --help        Print help
+       --state-dir   Path to store state information
+   -v, --verbose     Show all information messages
+       --version     Print version number
+
+``add``
+-------
+
+Generates a token for a new server
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster add <NAME> [flags]
+
+
+``bootstrap``
+-------------
+
+Sets up a new cluster
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster bootstrap [flags]
+
+``config``
+----------
+
+Manage Ceph Cluster configs
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster config [flags]
+  microceph cluster config [command]
+
+Available Commands:
+
+.. code-block:: none
+
+  get         Get specified Ceph Cluster config
+  list        List all set Ceph level configs
+  reset       Clear specified Ceph Cluster config
+  set         Set specified Ceph Cluster config
+
+
+``config get``
+--------------
+
+Get specified Ceph Cluster config
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster config get <key> [flags]
+
+
+``config list``
+---------------
+
+List all set Ceph level configs
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster config list [flags]
+
+
+``config reset``
+----------------
+
+Clear specified Ceph Cluster config
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster config reset <key> [flags]
+
+Flags:
```

Comment:
'Options:' everywhere

Reply:
Same as above, can we standardize on `[flags]`?

Reply:
Yep.